tests/testing_nurbs.py

```diff
@@ -258,6 +259,34 @@ def test_nurbs_couple_nurbs_meshes(self):
         # Compare with the reference file
         compare_test_result(self, input_file.get_string(header=False))
 
+    def test_create_nurbs_base_sphere_patch_surf(self):
+        """Test the creating of the base patch of the surface of a sphere."""
+
+        # Create input file
+        input_file = InputFile()
+
+        # Create the base of a sphere
+        surf_obj = create_nurbs_base_sphere_patch_surf(1, n_ele_u=3, n_ele_v=2)
+
+        # Create first patch set
+        mat = MaterialStVenantKirchhoff(youngs_modulus=430, nu=0.10, density=4.2e-7)
```

Comment:
You can leave this empty, as these values are not relevant for the test and the functionality of `MaterialStVenantKirchhoff` is already tested.